src/server/server_family.cc

```diff
@@ -129,6 +129,8 @@ ABSL_FLAG(bool, info_replication_valkey_compatible, true,
 
 ABSL_FLAG(bool, managed_service_info, false,
           "Hides some implementation details from users when true (i.e. in managed service env)");
+ABSL_FLAG(int32_t, max_squashed_cmd_num, 32,
+          "Max number of commands quashed in command squash optimizaiton");
```

Comment:
```suggestion
ABSL_FLAG(uint32_t, max_squashed_cmd_num, 32,
          "Max number of commands squashed in pipeline squash optimizaiton");
```

Reply:
this is not only for pipeline optimization this is why I did not add the pipeline word there.
I will fix the typo